src/Compilers/VisualBasic/Portable/Emit/SynthesizedPrivateImplementationDetailsSharedConstructor.vb

```diff
@@ -0,0 +1,84 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Generic
+Imports System.Collections.Immutable
+Imports System.Diagnostics
+Imports System.Linq
+Imports Microsoft.CodeAnalysis.CodeGen
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    Friend NotInheritable Class SynthesizedPrivateImplementationDetailsSharedConstructor
+        Inherits SynthesizedConstructorSymbol
```

Comment:
This is incorrect and should be SynthesizedGlobalMethodBase. That change will enable reverting the other changes that tolerate null containing types of methods.